conf/modules/adc_expansion_uart.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="adc_expansion_uart">
+  <doc>
+    <description>Get analog signals from expansion board to main autopilot over uart</description>
```

Comment:
Well, gen_modules could be improved, true...
But "simply" using something like
```
ADC expansion via UART. Get analog signals from expansion board to main autopilot over uart.
```
as description would have also done the job...